src/Stripe.net/Entities/V2/Events/Event.partial.cs

```diff
@@ -5,18 +5,29 @@ namespace Stripe.V2
     using System.Threading;
     using System.Threading.Tasks;
     using Newtonsoft.Json;
+#if NET6_0_OR_GREATER
+    using STJS = System.Text.Json.Serialization;
+#endif
+
     using Stripe.Infrastructure;
 
     /// <summary>
     /// Manually-maintained convenience methods added to ThinEvent.
     /// </summary>
     [JsonConverter(typeof(V2EventConverter))]
+#if NET6_0_OR_GREATER
+    [NoSystemTextJsonAttributesNeeded("Converter is only needed for deserialization")]
```

Comment:
I don't understand this message. Event is being deserialized, right?

Reply:
Not through STJ.  This PR only supports serializing with STJ, so any conversion logic needed to deserialize an API response into a StripeObject can be omitted.  In some cases, for testing purposes or because it was easy to do so, I implemented both sides of a converter; in this case I punted but I would be game to go ahead and implement this to remove this potential point of confusion.

Reply:
An update here: the V2EventConverter here is only used to deserialize the v2.core.event object into the specific event class as indicated by the type property.  After that, it's just an object and our existing attributes should be able to handle this.  However, in proving this to myself, I realized that this does not hold true for when we're converting a StripeList<V2.Event> because of how STJ supports converting items inside of a collection, so this does need a converter after all.